target/linux/ath79/image/generic.mk

```diff
@@ -49,6 +49,15 @@ define Device/glinet_ar300m_nor
 endef
 TARGET_DEVICES += glinet_ar300m_nor
 
+define Device/ligowawe_dlb5_propeller
+  ATH_SOC := ar9342
+  DEVICE_TITLE := LigoWawe DLB 5 Propeller
+  DEVICE_PACKAGES := rssileds uboot-envtools
+  SUPPORTED_DEVICES += ligowawe,dlb5-propeller dlb5-propeller
```

Comment:
There is an automatically generated SUPPORTED_DEVICES and it is generated by replacing the underline in the image name with a comma. If the image name above is defined correctly, this line is not needed.
The added second SUPPORTED_DEVICES, as you could probably see in other entries in this file, comes from ar71xx and is used to make image from ath79 pass ar71xx's sysupgrade firmware check. If this device isn't supported by ar71xx at all you needn't add that.

Reply:
Will drop it,I was testing it out and forgot to drop.